infra/sql/mysql/002_worker.sql

```diff
@@ -66,22 +66,19 @@ CREATE TABLE `job_result` (
   KEY `uniq_key_idx` (`uniq_key`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
 
--- runner definition
-DROP TABLE IF EXISTS runner_schema;
-CREATE TABLE `runner_schema` (
+-- worker schema definition
+DROP TABLE IF EXISTS worker_schema;
+CREATE TABLE `worker_schema` (
   `id` BIGINT(10) PRIMARY KEY,
   `name` VARCHAR(128) NOT NULL, -- name for display
   `operation_type` INT(10) NOT NULL, -- enum: command:0, httpRequest:2, grpcUnary:3, docker:4, plugin:1)
-  `operation_proto` MEDIUMTEXT NOT NULL, -- protobuf definition for the runner operation(main message must be named end with "Operation")
```

Comment:
remove operation_proto.
(use pre-defined only, plugin use only job_arg_proto for custom.)